app/lib/isNotEnglishPostcode.js

```diff
@@ -0,0 +1,8 @@
+// http://www.roblocher.com/technotes/uk-postareas.html
+const excludedRegEx = /^(AB|BT|CF|DD|EH|FK|G|GY|HS|IM|IV|JE|KA|KW|KY|ML|PA|PH|SA|ZE)\d+/i;
```

Comment:
Based on the conversation /w Dean, @ermlikeyeah, @neilbmclaughlin about how this also changes the behaviour of the application - removing this will push the user to have a `no results` screen, instead of defaulting to the `this is not an English postcode` screen and we are trying to be helpful.

I agree that once the country/countries are in place we should remove this code, so raising this as an issue here: https://github.com/nhsuk/gp-finder/issues/146